app/src/main/java/edu/temple/templetag/fragments/TagListFragment.java

```diff
@@ -0,0 +1,65 @@
+package edu.temple.templetag.fragments;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import edu.temple.templetag.R;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link TagListFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class TagListFragment extends Fragment {
```

Comment:
Rename to TagRecyclerViewFragment and add a TagRecyclerViewAdapter like in the UML if possible

Reply:
I have not worked on it yet. I have to coordinate with Zach cuz he assigns this task to himself.